smac/optimizer/ei_optimization.py

```diff
@@ -195,8 +195,8 @@ def _maximize(
 
         """
 
-        num_configurations_by_local_search = self._calculate_num_points(
-            num_points, stats, runhistory
+        num_configurations_by_local_search = min(
```

Comment:
Disclaimer: I do not really understand why this is the same as before, but I believe this could be true.

Reply:
I don't get your question - but what's happening here is that I actually remove the heuristic in `_calculate_num_points`.

Reply:
That wasn't a question, rather a remark that it is not clear to me why this code is changed.